UnityProject/Assets/Scripts/Messages/Server/TileChangesNewClientSync.cs

```diff
@@ -16,7 +16,7 @@ public struct NetMessage : NetworkMessage
 		}
 
 		//just a best guess, try increasing it until the message exceeds mirror's limit
-		private static readonly int MAX_CHANGES_PER_MESSAGE = 20;
+		private static readonly int MAX_CHANGES_PER_MESSAGE = 100;
```

Comment:
Wasnt this reduced due to length issues?

Reply:
looking at the git history was always 10, since the start the offer we think of tile change manager that uses a different message

Reply:
./Sprite change message

Reply:
ah ok